packages/lit-html/src/lit-html.ts

```diff
@@ -328,17 +337,25 @@ class Template {
     let node: Node | null;
     let nodeIndex = 0;
     let bindingIndex = 0;
+
+    // If we're in attribute value position, this will be the index of the end
+    // of the attribute name.
```

Comment:
Comment confuses this with `attrNameEndIndex`

Reply:
This is the index into the `attrNames` array, right?

Reply:
Oops, I added this to the wrong thing